include/revng/ABI/Definition.h

```diff
@@ -145,6 +145,13 @@ type: struct
       ARM ABI), the value of `StackAlignment` should be equal to 4.
     type: uint64_t
 
+  - name: MinimumStackWordSize
+    doc: |
+      States the minimum possible stack argument size in bytes.
+      When the factual type of an argument is below this value, it should be
```

Comment:
factual?

Reply:
Was supposed to be `actual`, but I suppose I was thinking about some kind of `facts` at the time 😆